### PR TITLE
fix: missing translations

### PR DESCRIPTION
**Pull request by FI00ds**
# Pull Request

## Description

* Added missing translations for optimiser tab
* Modified stat weight sliders and E/S placeholders to adjust width as necessary

## Related Issue

* 

## Checklist

- [x] I have added commit messages that are descriptive and meaningful.
- [x] I have tested the changes locally.
- [x] I have reviewed the code changes.

### CODE REVIEW

**Comment by fribbels** [comment]
I think the right CSS for the button previews is:

image

Parent: `width: 'max-content'`
Children: `flex: 'auto'`

File: src/components/optimizerTab/optimizerForm/FormStatRollSlider.tsx [hunk 1]
```diff
@@ -1,38 +1,71 @@
...
-            onChangeComplete={(x) => window.onOptimizerFormValuesChange(x, window.optimizerForm.getFieldsValue(), true)}
-          />
-        </Form.Item>
+      <Flex
+        vertical
+        style={
+          typeof navigator !== 'undefined' && navigator.userAgent.toLowerCase().indexOf('firefox') > -1
+            ? { width: '-moz-available' }
+            : { width: '-webkit-fill-available' }
+        }
```
Comment (fribbels) [inline src/components/optimizerTab/optimizerForm/FormStatRollSlider.tsx:65]: This looks like pretty bad practice/unnecessary, is there a reason this isn't just `width: '100%'`?

Comment (FI00ds) [reply src/components/optimizerTab/optimizerForm/FormStatRollSlider.tsx:65]: that led to the slider clipping into the text in chinese, and trying other methods the sliders just shrank down to 0 width (or close to it)

Comment (fribbels) [reply src/components/optimizerTab/optimizerForm/FormStatRollSlider.tsx]: I removed the fixed width and gave it fixed margins instead
```
          width: '100%',
          marginLeft: 10,
          marginRight: 10,
```
instead, looks like that should be good enough

**Pull request merged**